sdl_android/src/main/java/com/smartdevicelink/transport/SdlRouterService.java

```diff
@@ -1291,8 +1291,8 @@ public boolean writeBytesToTransport(Bundle bundle){
 			if(mSerialService !=null && mSerialService.getState()==MultiplexBluetoothTransport.STATE_CONNECTED){
 				byte[] packet = bundle.getByteArray(TransportConstants.BYTES_TO_SEND_EXTRA_NAME); 
 				int offset = bundle.getInt(TransportConstants.BYTES_TO_SEND_EXTRA_OFFSET, 0); //If nothing, start at the begining of the array
```

Comment:
We should probabaly move this line under the `packet!=null` check as well since it doesn't make sense to allocate this if the packet is null.